### ISSUE TITLE
train loss didn't change after 15k epoch

### ISSUE THREAD

**Issue by elgiano** [moiseshorta/RAVE-Latent-Diffusion#2]
Hi and thanks for this, it looks super interesting and I can't wait to try it out :)
I'm trying to train a model, but after 15k epochs the loss didn't go down a tiny bit (train loss ~2.008, val loss ~1.97) .. is it expected or something is not right? the rave model I'm using is trained with a v2 config, the dataset is around 10 hours of records from a single artist, although quite  heterogeneous.

**Comment by elgiano** [comment]
sorry, my bad, it was because unlike rave, preprocess doesn't load files recursively and doesn't load anything else than wavs. I'm making a PR to add these two features, if you'd like to have them :)
now that I loaded the dataset correctly, loss is going down and it's looking good <3
